### ISSUE TITLE
Is the im2latex-100k-dataset the same as that provided by the original paper

### ISSUE THREAD

**Issue by Maggione** [OpenNMT/OpenNMT-py#672]
Is the dataset from http://lstm.seas.harvard.edu/latex/im2text.tgz the same as that provided by the original paper? I followed the script, but can not reproduce the result in [Image-to-Markup Generation with Coarse-to-Fine Attention](https://arxiv.org/pdf/1609.04938.pdf).

**Comment by da03** [comment]
Yes the dataset is the same. However, the image encoder presented in OpenNMT-py is not the same as original im2markup (https://github.com/harvardnlp/im2markup), because here we assumed RGB images to make our image encoder general, while in the original im2markup we assumed greyscale images which has fewer parameters.

**Comment by srush** [comment]
@da03 do you have the numbers with OpenNMT-py?

**Comment by da03** [comment]
The training set and validation set are smaller because we dropped large images and long formulas, while we keep test set untouched, and this is what we used for training the models in the paper. The unprocessed (and processed) dataset can be found here: http://lstm.seas.harvard.edu/latex/data/

**Comment by Maggione** [comment]
I see, thank you so much!

**Comment by da03** [comment]
No problem!
